utils/debug/GeometryDebug.js

```diff
@@ -0,0 +1,44 @@
+export default {
+    applyToNodeFirstMaterial(view, root, layerId, cb) {
```

Comment:
and you won't need this function any more. Or maybe only for wireframe, though I'm fine with having `feature.material.wireframe = layer.wireframe` in WFS_Processing.update. This is a useful feature for client apps too.